lib/main.dart

```diff
@@ -4,34 +4,16 @@ import "package:flutter/material.dart";
 import "package:flutter_blue/flutter_blue.dart";
 import "package:flutter_gen/gen_l10n/app_localizations.dart";
 
-void main() {
-  runApp(const MyApp());
-}
+void main() => runApp(const MyApp());
 
 class MyApp extends StatelessWidget {
-  const MyApp({super.key});
+  const MyApp({Key? key}) : super(key: key);
```

Comment:
```suggestion
  const MyApp({super.key});
```
Why remove the shorthand syntax?